packages/eslint-plugin/tests/rules/no-duplicate-imports.test.ts

```diff
@@ -0,0 +1,109 @@
+import rule from '../../src/rules/no-duplicate-imports';
+import { RuleTester } from '../RuleTester';
+
+const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
+});
+
+ruleTester.run('no-dupe-class-members', rule, {
```

Comment:
Hi :)
Isn't this rule id should be `no-duplicate-imports`, not `no-dupe-class-members`?